docs/userguide/storagedriver/imagesandcontainers.md

```diff
@@ -148,15 +142,6 @@ One way to do this is using a Dockerfile and the `docker build` command.
     The 4 image layers below it are the exact same image layers the make up the
     ubuntu:15.04 image as their UUIDs match.
```

Comment:
I vote that we call a code freeze on the whole thing..... progress is overrated ;-)

Just kidding..... I just need some time look into it and get my head around the changes. Once I've done that I think the entire storage driver document set probably needs reviewing and revalidating. We also need a way to reflect the differences between older engines and those with the new stuff. I'll summarize my thoughts on that to @moxiegirl